lib/v2/biostars/herald.js

```diff
@@ -0,0 +1,42 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const homeUrl = 'https://www.biostars.org/t/herald';
```

Comment:
Please make it support custom tag input like `/tag/transcriptome/`, `/tag/RNAseq/`, `/tag/workshop/` or `/tag/transcriptomics/` instead of a single hard-coded tag only.

Reply:
Thanks for your suggestions.
I create a new PR at https://github.com/DIYgod/RSSHub/pull/13748#issue-1987638158.